### PR TITLE
Cart Management

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Cart Management Functionality Added.
- Added Add_To_Cart func with using Cart API.
  =>  RemoveFromCart using API & Inc | Dec Cart Quantity.
- Updated nav badges when item Added to cart.
- Added Cart Billing section with Funcs.
   CartBilling => TotalAmount , CouponDiscount, Total-items-in Cart . 
- refactored: some CSS too.
image

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]
- code looks clean
- i like your ui
- context is working fine

File: Zero-Gravity-Store-React/src/Pages/AuthWidget/Login/Login.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ export function Login() {
         email: formState.email,
         password: formState.password,
       });
-      console.log(userData);
-      localStorage.getItem("token", userData.data.encodedToken);
+      console.log(authState.token);
```
Comment (Rohitprasad83) [inline Zero-Gravity-Store-React/src/Pages/AuthWidget/Login/Login.js:50]: remove console.logs

File: Zero-Gravity-Store-React/src/Pages/CartWidget/Cart.css [hunk 2]
```diff
@@ -22,7 +22,7 @@
     height: 24rem;
     position: relative;
     color: var(--card-text);
-    padding: 0 23rem;
+    /* padding: 0 23rem; */
```
Comment (Rohitprasad83) [inline Zero-Gravity-Store-React/src/Pages/CartWidget/Cart.css:25]: remove unwanted code

File: Zero-Gravity-Store-React/src/Pages/CartWidget/Components/Mycart.js [hunk 3]
```diff
@@ -1,67 +1,54 @@
...
+                {cartState.cartProducts.map((product) => {
+                  return <CartProducts key={product._id} product={product} />;
+                })}
               </div>
+              <CartInvoice />
             </div>
+          </main>
+        ) : (
+          <div className="empty-cart">
+            {/* <img className="empty-cart-img" src={emptyCart} alt="image" /> */}
```
Comment (Rohitprasad83) [inline Zero-Gravity-Store-React/src/Pages/CartWidget/Components/Mycart.js:33]: remove commented code

**Review by Rohitprasad83 — APPROVED** [review]
Code looks clean, just remove the unwanted code and console.logs

**Pull request merged**